telekom-workflow-web/src/main/resources/META-INF/resources/WEB-INF/jsp-twe/console/workflow/instance.jsp

```diff
@@ -33,6 +33,7 @@
 	                    	<th><spring:message code="workflowinstance.refnum"/></th>
 	                    	<th><spring:message code="workflowinstance.nameandversion"/></th>
 	                    	<th><spring:message code="workflowinstance.keephistory"/></th>
+							<th><spring:message code="workflowinstance.archivePeriodLength"/></th>
```

Comment:
browser shows that the indentation tabs/spaces are probably different than on other rows

Reply:
:exploding_head: didn't show it in IDEA, but it seems like these rows have spaces + tabs mixed. Fixed it now by using exact same whitespaces as other rows.